src/soc/ibm/power9/Makefile.inc

```
@@ -0,0 +1,12 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+ifeq ($(CPU_POWER9),y)
+
+bootblock-y += bootblock.c
+postcar-y	+= exit_car.S
```

```suggestion
postcar-y += exit_car.S
```
Single space everywhere.

---

Renamed it